packages/entities/entities-upstreams-targets/src/composables/useHelpers.ts

```diff
@@ -120,10 +120,6 @@ export default function useHelpers() {
       payload.healthchecks.active?.healthy?.interval !== undefined)
       ? payload.healthchecks.active?.healthy?.interval?.toString()
       : '0'
-    result.activeHealthCheck.interval = (payload.healthchecks.active?.healthy?.interval !== null &&
-      payload.healthchecks.active?.healthy?.interval !== undefined)
-      ? payload.healthchecks.active?.healthy?.interval?.toString()
-      : '0'
```

Comment:
`activeHealthCheck.interval` is already assigned in the lines [above](https://github.com/Kong/public-ui-components/blob/27882af690aa6dd668062597ba3570eaf539b724/packages/entities/entities-upstreams-targets/src/composables/useHelpers.ts#L119-L122)